string atom
float32 prob
